src/Redux/books/books.js

```diff
@@ -0,0 +1,59 @@
+const ADD_BOOK = 'books/ADD_BOOK';
+const REMOVE_BOOK = 'books/REMOVE_BOOK';
+// const GET_BOOKS = 'books/GET_BOOKS';
+
```

Comment:
- [ ] Kindly remove all unnecessary commented-out code in this file. Removing such codes makes your code clean and it shows that you adopt best JavaScript practices when building an application. 👍